features/step_definitions/up.rb

```diff
@@ -0,0 +1,9 @@
+
+Then(/I should have (.+) installed/) do |tool_path|
+    @tool_path = tool_path
```

Comment:
We don't have the convention enforced by any linter yet but I'd stick to the two spaces indentation.

Reply:
I've opened an issue to add the linter https://github.com/tuist/tuist/issues/211